### PR TITLE
feat: added avatar and badge components

### PR DESCRIPTION
**Pull request by sharath-io**
Avatar component: 

Screen Shot 2022-02-10 at 11 12 06 PM
Screen Shot 2022-02-10 at 11 12 15 PM

Badge component:

Screen Shot 2022-02-10 at 11 12 24 PM
Screen Shot 2022-02-10 at 11 12 27 PM

### CODE REVIEW

**Comment by rishbitsnbytes** [comment]
Everything looks good to me!!

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+:root {
```
Comment (chanchal16) [inline styles.css:1]:
```
Suggestion: you can use a separate file for variables. The rest looks good.
```

**Review by chanchal16 — APPROVED** [review]

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sonishreyas) [inline components/avatar/avatar.html:8]: You can add a meta author tag here later

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      <img class="avatar size-s" src="https://i.pravatar.cc/400?img=65">
+      <img class="avatar size-m" src="https://i.pravatar.cc/400?img=13">
+      <img class="avatar size-l" src="https://i.pravatar.cc/400?img=2">
+    </div>
+    <p class="description">Add the avatar class to img element.Use the class avatar for avatar component and classes
+      size-s size-m size-l for avatars of different sizes</p>
+
+    <iframe
+      src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=500&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=38px&ph=59px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=162%25&si=false&es=2x&wm=false&code=%2520%253Cimg%2520class%253D%2522avatar%2520size-s%2522%2520src%253D%2522%2520%2522%253E%250A%2520%253Cimg%2520class%253D%2522avatar%2520size-m%2522%2520src%253D%2522%2520%2522%253E%250A%2520%253Cimg%2520class%253D%2522avatar%2520size-l%2522%2520src%253D%2522%2520%2522%253E"
+      style="width: 460px; height: 213px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (sonishreyas) [inline components/avatar/avatar.html:65]: You can add a class to iframes so that they can become responsive. This can be done later if you need

**Review by sonishreyas — APPROVED** [review]
I have kept a few suggestions you can do later on. Rest looks good to me

File: components/avatar/avatar.html (hunk 2 above)
Comment (sharath-io) [reply components/avatar/avatar.html:8]: Added meta author tag. Thank you for suggestion

File: components/avatar/avatar.html (hunk 3 above)
Comment (sharath-io) [reply components/avatar/avatar.html:65]: Will update this later for responsiveness. Thank you for suggestion

File: styles.css (hunk 1 above)
Comment (sharath-io) [reply styles.css:1]: Will add variable css file later, while file restructuring. Thank you for suggestion

**Pull request merged**